### PR TITLE
Added the .gitignore file

### PR DESCRIPTION
**Pull request by mik-dass**
fabric8-hdd-bug-id: fabric8-hdd/supervisor#8
Signed-off-by: mik-dass mrinald7@gmail.com

### CODE REVIEW

**Comment by anmolbabu** [comment]
This PR in its current form looks good to me.
We can evolve this as need arises
Approving and merging for now

**Review by anmolbabu — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/node_modules/
```
Comment (anmolbabu) [inline .gitignore:1]: Should this be /node_modules or ./node_modules ?

**Review by anmolbabu — CHANGES_REQUESTED** [review]
One small comment

File: .gitignore (hunk 1 above)
Comment (mik-dass) [reply .gitignore:1]: /node_modules/ worked for me, and I referred this article : https://www.atlassian.com/git/tutorials/gitignore
and it seemed alright to me.
 @anmolbabu @miteshvp @dev-gaur  Correct me if I am wrong

Comment (anmolbabu) [reply .gitignore:1]: Ack

**Review by anmolbabu — APPROVED** [review]
Merging based on https://github.com/fabric8-hdd/supervisor/pull/14/files#r155157041

**Pull request merged**